l/librdkafka/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+   "maintainer": "Balavva-Mirji",
+   "package_name": "librdkafka",
+   "github_url": "https://github.com/confluentinc/librdkafka",
+   "version": "v2.5.3",
+   "default_branch": "master",
+   "build_script": "lidbrdkafka-2.5.3_ubi_9.3.sh",
+   "package_dir": "l/librdkafka",
+   "docker_build": false,
+   "validate_build_script": true,
+   "use_non_root_user": false,
+   "librdkafka-*.*.*-*": {
```

Comment:
The regex needs to be for the version. So in this case it will be v2.*.*. or v2.5.* if it is specific for 2.5.x versions.

Reply:
Done with requested changes.